index.js

```diff
@@ -609,12 +609,10 @@ function gitHubPull() {
 }
 
 function gitHubCommit(commitMessage) {
+    // todo: only do commit when there is an actual change in the editor
     if (isRepoDefined()) {
-        if (!commitMessage || commitMessage === null) {
+        if (!commitMessage) {
             commitMessage = prompt("Enter commit message:");
-        }
-        if (commitMessage === null) {

```

Comment:
The check for the commit message being null after the prompt is necessary to allow the user to cancel the commit, but still allow a portal based save - please add again.